### DISCUSSION TITLE
My Personal Team is still paused, but I moved the limit-exceeded project to another one

### DISCUSSION THREAD

**Discussion by shutock** [vercel/community#6634] [Account Issue]
### Question

One of my projects exceeded the hobby plan, so I moved it to a different team and upgraded to a Pro plan. Everything is fine with this project. However, my main hobby team still has a "paused" label next to its name. Your support AI tell my that I need to contact the Support, so I'm here

**Comment by amyegan** [comment]
Hey @shutock. Please [submit a case](https://vercel.com/help) to the Customer Success Team for help un-pausing your team
